ferrocene/doc/qualification-plan/src/history.rst

```diff
@@ -112,7 +112,7 @@ Contributing to Upstream
 
 A developer may contribute to upstream by either requesting a feature to be
 developed, reporting a bug, submitting a Pull Request for review and merging into
-the main branch, or writing documentation.
+the master branch, or writing documentation.
 
```

Comment:
I would instead use the text "and getting it merged", because the name of the branch not really relevent (and may get changed in the future, as it should)